modules/va_facilities/app/controllers/va_facilities/docs/metadata_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module VaFacilities
+  module Docs
+    class MetadataController < ::ApplicationController
+      skip_before_action(:authenticate)
+
+      def index
+        render json: {
+          meta: [
+            {
+              version: '1.0.0',
+              internal_only: false,
+              status: VERSION_STATUS[:dra],
+              source: 'https://api.va.gov/services/va_facilities/docs/v1/api',
+              dev_source: 'https://dev-api.va.gov/services/va_facilities/docs/v1/api',
```

Comment:
Given that 9 times out of 10 our docs are the same across the various envs, do you think we need to list it out like this?

Reply:
We need a way to seperate the versions that show up on different docs based on environment, we could split it up between base and path and let developer.va.gov pick which base to use but still need to be able to split it up.

Reply:
Switched it to path: instead of url source so that the app can determine the base route.